tracer/src/tracer.rs

```diff
@@ -134,16 +140,17 @@ impl TraceBlock {
 
 impl Drop for TraceBlock {
     fn drop(&mut self) {
+        // SAFETY: start is a variable that's local to the current thread
+        let start = unsafe { TRACER.get().unwrap().start };
```

Comment:
`// SAFETY: start has been initialised as part of initialising the value of TRACER`